lib/child_process.js

```diff
@@ -396,85 +391,85 @@ function normalizeSpawnArguments(file, args, options) {
     args = [];
   }
 
-  if (options === undefined)
+  if (options === undefined) {
     options = {};
-  else if (options === null || typeof options !== 'object')
+  } else if (options === null || typeof options !== 'object') {
     throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
                                'options',
                                'object',
                                options);
-
-  // Validate the cwd, if present.
-  if (options.cwd != null &&
+  } else {
```

Comment:
If I'm not wrong this branch does not add any real benefit, it only increase indentation.

Reply:
This is a performance optimization for the case that `options` is undefined. Right now, it will be set to an empty object and then tests multiple entries. That is unnecessary and can be prevented by just using else here.